aiidalab_widgets_base/utils/exceptions.py

```diff
@@ -0,0 +1,16 @@
+class AtLeastTwoStepsError(ValueError):
+    """Raised when the number of steps is less than two."""
```

Comment:
Would be good to have a more thorough description here, it's not clear that this exception is for the WizardAppWidget. Which makes me think, since this exception is rather specific, it would make sense to put it in the same file as the widget itself imo.

Reply:
I also thought about this, good that you've mentioned it. Agreed, moved to `wizard..py`.